docs/examples_benchmark.md

```diff
@@ -0,0 +1,68 @@
+# Real-world Example Benchmarks
+For all three real-word examples (PNEUMONIA, NER, CCFRAUD), we perform benchmark testing to assess three main aspects: the training overhead, model performance, and scalability of training under FL setting. The main purpose of this benchmark is to show that FL has been implemented correctly and works as expected, rather than to focus on the exact metric value shown below, which will vary based on distinct models and datasets. In the following experiments, each silo is trained with the same hyperparameters to ensure the consistency of results, and the model performance was assessed with all test data combined from silos. Below are the results for each aspect.
+
+## Table of contents
+
+- [Training Overhead](#real-world-example-benchmarks)
+- [Model Performance](#model-performance)
+- [Scalability with Training](#scalability-with-training)
+
```

Comment:
I would add a methodology section to explain some of the context of the benchmark:

maybe a table with each problem : CCFRAUD, NER, PNEUMONIA
and rows giving some of the metadata, such as.

- problem: classification, regression, ...
- type of data: image vs text vs tabular
- number of samples
- size on disk

Reply:
in the methodology, you could also spend some time defining each of the variants: centralized-1, centralized-1/3 and FL.

Reply:
you can characterize each variant (c-1, c1/3, FL) and explain the number of silos, the SKUs, and the regions for FL